src/components/Table/GridTable.tsx

```diff
@@ -1349,10 +1345,6 @@ export function matchesFilter(maybeContent: ReactNode | GridCellContent, filter:
   return false;
 }
 
-function maybeDarken(color: string | undefined, defaultColor: string): string {
```

Comment:
I'm personally a little skeptical that a single hover color will end up working long-term, and I thought `maybeDarken` was pretty cute :-), but I get that we'd probably use like a configured-per-level style going forward, so +1.